packages/features/tasker/tasks/crm/schema.ts

```diff
@@ -0,0 +1,5 @@
+import z from "zod";
+
+export const createCRMEventSchema = z.object({
```

Comment:
Ideally we only pass the bookingUid from `handleNewBooking` to the tasker. This helps decouple the two services.